plugins/processors/port_name/services_path_notwin.go

```diff
@@ -0,0 +1,7 @@
+// +build !windows
```

Comment:
> A foolish consistency is the hobgoblin of little minds

With that in mind, let's rename this file to either `services_path_notwindows.go`, or `services_path_posix.go`

Reply:
No problem.  I renamed it _notwindows